mkosi/initrd.py

```diff
@@ -169,6 +170,36 @@ def main() -> None:
 
         cmdline += ["--sandbox-tree", sandbox_tree]
 
+        # Generate fstab with all the x-initrd.mount entries
+        if Path("/etc/fstab").exists():
+            fstab = []
+            for line in textwrap.dedent(Path("/etc/fstab").read_text()).splitlines():
+                entry = line.strip().split()
+                if len(entry) < 4 or entry[0].find("#") >= 0 or entry[3].find("x-initrd.mount") == -1:
+                    continue
+                fstab += [line]
```

Comment:
The split already removes the whitespace, so the strip is not necessary
```suggestion
                entry = line.split()
                if len(entry) < 4 or entry[0].startswith("#") or "x-initrd.mount" in entry[3]:
                    continue
                fstab.append(line)
```